src/fn/upload-app-settings.js

```diff
@@ -11,7 +12,19 @@ module.exports = {
       .then(json => {
         // As per https://github.com/medic/medic-webapp/issues/3674, this endpoint
         // will return 200 even when upload fails.
-        if(!json.success) throw new Error(json.error);
+        if (!json.success) {
+          throw new Error(json.error);
+        }
+
+        if ('updated' in json) {
+          // the `updated` param was added in 3.9
+          // https://github.com/medic/cht-core/issues/6315
+          if (!json.updated) {
+            warn('Settings not updated - no changes detected');
```

Comment:
IMO this should just be an info as we don't know that anything has gone wrong. This would be consistent with other places that we're detecting no update, eg: https://github.com/medic/medic-conf/blob/301-check-properties-too/src/lib/upload-forms.js#L68

Reply:
Done!